### ISSUE TITLE
Bold text that uses stars gets converted to a dot point

### ISSUE THREAD

**Issue by Dan503** [hosseinmd/prettier-plugin-jsdoc#91]
If I have the following comment:

```
    /**
     * Some comment text.
     *
     * **Warning:** I am a warning.
     */
```
It should get rendered like this

> Some comment text.
>
> **Warning:** I am a warning.

However prettier converts it to this:

```
    /**
     * Some comment text.
     *
     * - *Warning:** I am a warning.
     */
```

Which gets rendered as

> Some comment text.
>
> - \*Warning:*\* I am a warning.

**Comment by Dan503** [comment]
So far I've been working around this issue by formatting it like this:

```
    /**
     * Some comment text.
     *
     * __Warning:__ I am a warning.
     */
```

It works but I would proffer if I could use stars for bold text rather than underscores.

**Comment by hosseinmd** [comment]
Published in v0.3.12
